### ISSUE TITLE
Death to the Pop-Ups

### ISSUE THREAD

**Issue by kylebrandt** [bosun-monitor/bosun#1099]
Bosun will throw some pop-ups in the following two instances (and you often need to click "stop showing additional warnings..."

Both cases happen on the dashboard

 1. Connection no longer available to Bosun (i.e. you were on a VPN, and now you are not)
 2. Something has been forgotten, but this hasn't been reflected on the dashboard. So when you expand the group pops complain about the alert key no longer existing.

Basically I think it would be better to show these errors on the page, and not a popup.

**Comment by kylebrandt** [comment]
Haven't seen this in a long time, somehow it must have been fixed.
